zuul.d/architecture-jobs.yaml

```diff
@@ -49,13 +49,6 @@
     nodeset: centos-9-medium-3x-centos-9-crc-extracted-2-39-0-3xl-vexxhost
     vars:
       <<: *architecture_vars
-      cifmw_architecture_automation_file: >-
```

Comment:
There will probably be some more work as well on that front:
```
scenarios/reproducers/bgp-4-racks-3-ocps.yml:cifmw_architecture_automation_file: >-
scenarios/reproducers/bgp-4-racks-3-ocps.yml-  {{
scenarios/reproducers/bgp-4-racks-3-ocps.yml-    (ansible_user_dir,
scenarios/reproducers/bgp-4-racks-3-ocps.yml-     'src/github.com/openstack-k8s-operators',
scenarios/reproducers/bgp-4-racks-3-ocps.yml-     'architecture/automation/vars',
scenarios/reproducers/bgp-4-racks-3-ocps.yml-     cifmw_arch_automation_file) |
scenarios/reproducers/bgp-4-racks-3-ocps.yml-     path_join
scenarios/reproducers/bgp-4-racks-3-ocps.yml-  }}
```

Guess some cleaning is mandatory :).

Reply:
Done